Clarify that you should put all cluster members in list of discovery endpoints (#802)
neo4j/docs-operations: modules/ROOT/pages/configuration/configuration-settings.adoc

[frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
|===
|Description

> a|A comma-separated list of endpoints which a server should contact in order to discover other cluster members. Typically you should put all members of your cluster, including the current server, in this list.

Reviewer: ```suggestion
a|A comma-separated list of endpoints which a server should contact in order to discover other cluster members. 
Typically, all members of the cluster, including the current server, should be specified in this list.
```
